apps/beeai-ui/src/components/AppHeader/AppHeader.tsx

```diff
@@ -43,10 +43,14 @@ export function AppHeader({ className }: Props) {
           <MainNav />
 
           {NAV.length > 0 && <AppHeaderNav items={NAV} />}
+
           {!NAV.length && agent && (
             <>
               <p className={classes.agentName}>{getAgentDisplayName(agent)}</p>
-              <AgentDetailButton />
+
+              <div className={classes.agentDetailButton}>
```

Comment:
The name `.agentDetailButton` suggests it refers to the **button** itself, not a wrapper or container. A better name might be `.agentDetailButtonContainer`.